src/domains/ecommerce-app/api/ecommerce-checkout/v1/_post_sessions.xml.tpl

```diff
@@ -0,0 +1,19 @@
+<policies>
+    <inbound>
+    <rate-limit-by-key 
+    calls="150"
```

Comment:
Isn't the threshold of 150 calls per 10 seconds too high?

Reply:
We set this threshold together with @infantesimone reasoning about 2 points:

1. this is the same rate limit set in the base policy for payment-request verify
2. thinking about one office with a proxy to go out in internet that expose same ip for all machines

Considering these 2 points we decide to set this value for this threshold